tests/regression_tests/benchmark/configs/sarcos.gbm.yaml

```diff
@@ -107,5 +107,3 @@ trainer:
   num_boost_round: 382
   num_leaves: 434
 model_type: gbm
-backend:
-  type: local
```

Comment:
Is there a reason we're removing backend as local? Should we potentially consider setting it to Ray?

Reply:
I believe default is ray